spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/client/GrpcClientProperties.java

```diff
@@ -39,23 +36,16 @@
 import io.grpc.netty.NettyChannelBuilder;
 
 @ConfigurationProperties(prefix = "spring.grpc.client")
-public class GrpcClientProperties implements EnvironmentAware {
+public class GrpcClientProperties {
```

Comment:
Note we still keep all the config props named channel variant here which is needed for type-safe config props, auto-completion in IDE, etc.. We just map them to the NamecChannel records in core (registry)